api/odl.py

```diff
@@ -788,22 +758,31 @@ class ODLXMLParser(OPDSXMLParser):
     NAMESPACES = dict(OPDSXMLParser.NAMESPACES,
                       odl="http://opds-spec.org/odl")
 
-class ODLBibliographicImporter(OPDSImporter):
-    """Import bibliographic information and formats from an ODL feed.
+class ODLImporter(OPDSImporter):
+    """Import information and formats from an ODL feed.
 
     The only change from OPDSImporter is that this importer extracts
     format information from 'odl:license' tags.
     """
-    NAME = ODLWithConsolidatedCopiesAPI.NAME
+    NAME = ODLAPI.NAME
     PARSER_CLASS = ODLXMLParser
 
     @classmethod
-    def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None):
+    def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None, do_get=None):
+        do_get = do_get or Representation.cautious_http_get
```

Comment:
Do you have a reason to be cautious here?

Reply:
I wanted to be able to pass `do_get` into the parser constructor from a test, but it already had a `do_get`, so I used that. The regular OPDSImporter does need to be cautious, and since I'm using the same one that's what normally gets used here. I can't use `HTTP.get_with_timeout` as the default since it's return values are different. I don't think the cautiousness hurts anything.

Reply:
You're right, it doesn't hurt anything, I just wanted to understand why you chose that here when you're not getting anything from Google Books or Gutenberg. It makes sense that you'd want to do things the same way as OPDSImporter.